client/header/activity-panel/panels/help.js

```diff
@@ -0,0 +1,290 @@
+import { recordEvent } from 'lib/tracks';
+import { getPaymentMethods } from 'task-list/tasks/payments/methods';
+import { compose } from 'redux';
+
+function getAppearanceItems() {
+	return [
+		{
+			title: __( 'Showcase your products and tailor your shopping experience using Blocks', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/woocommerce-blocks/?utm_source=help_panel',
+		},
+		{
+			title: __( 'Manage Store Notice, Catalog View and Product Images', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/woocommerce-customizer/?utm_source=help_panel',
+		},
+		{
+			title: __( 'How to choose and change a theme', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/choose-change-theme/?utm_source=help_panel',
+		},
+	];
+}
+
+function getPaymentsItems( props ) {
+	const { countryCode, profileItems } = props;
+	const paymentMethods = props.getPaymentMethods( {
+		activePlugins: [],
+		countryCode,
+		options: {},
+		profileItems,
+	} );
+
+	const methodIsVisible = ( methodKey ) => Boolean( paymentMethods.find( method => method.key === methodKey ) );
+
+	const showWCPay = methodIsVisible( 'wcpay' );
+	const showStripe = methodIsVisible( 'stripe' );
+	const showKlarnaCheckout = methodIsVisible( 'klarna_checkout' );
+	const showKlarnaPayments = methodIsVisible( 'klarna_payments' );
+	const showPayPal = methodIsVisible( 'paypal' );
+	const showSquare = methodIsVisible( 'square' );
+	const showPayFast = methodIsVisible( 'payfast' );
+
+	return [
+		{
+			title: __( 'Which Payment Option is Right for Me?', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/premium-payment-gateway-extensions/?utm_source=help_panel',
+		},
+		showWCPay && {
+			title: __( 'WooCommerce Payments Start Up Guide', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/payments//?utm_source=help_panel',
+		},
+		showWCPay && {
+			title: __( 'WooCommerce Payments FAQs', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/documentation/woocommerce-payments/woocommerce-payments-faqs/?utm_source=help_panel',
+		},
+		showStripe && {
+			title: __( 'Stripe Setup and Configuration', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/stripe/?utm_source=help_panel',
+		},
+		showPayPal && {
+			title: __( 'PayPal Checkout Setup and Configuration', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/paypal-express-checkout/?utm_source=help_panel',
+		},
+		showSquare && {
+			title: __( 'Square - Get started', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/woocommerce-square/?utm_source=help_panel',
+		},
+		showKlarnaCheckout && {
+			title: __( 'Klarna - Introduction', 'woocommerce-admin' ),
+			link: 'https://docs.woocommerce.com/document/klarna-checkout/?utm_source=help_panel',
+		},
+		showKlarnaPayments && {
+			title: __( 'Klarna - Introduction', 'woocommerce-admin' ),
```

Comment:
Would it be helpful to differentiate between these two `Klarna - Introduction`?

Reply:
Klarna Checkout was missed in the issue description, but I found it in the payments step. The two Klarna gateways are shown to different countries, so it's fine IMO that they have the same title.